deploy/charts/rawfile-csi/templates/01-controller-plugin.yaml

```
@@ -42,8 +42,9 @@ spec:
           image: "{{ .Values.controller.image.repository }}:{{ .Values.controller.image.tag }}"
           imagePullPolicy: {{ .Values.controller.image.pullPolicy }}
           args:
-            - csi-driver
-            - --disable-metrics
+            {{ range .Values.controller.csi_driver_args }}
+            - {{ . }}
+            {{ end }}
```

This is more idiomatic:

```suggestion
            {{- if .Values.controller.csi_driver_args -}}
            {{ .Values.controller.csi_driver_args | toYaml | nindent 10 }}
            {{- end }}
```

Also, prefer `.Values.controller.csiDriverArgs`, which is also more idiomatic